cmd/checkcommits/checkcommits.go

```diff
@@ -102,19 +112,28 @@ func checkCommitSubject(config *CommitConfig, commit, subject string) error {
 		return fmt.Errorf("Commit %v: pure whitespace subject", commit)
 	}
 
-	subsystemPattern := regexp.MustCompile(`^[^ ][^ ]*.*:`)
+	subsystemPattern := regexp.MustCompile(`^[[:blank:]]*([^:[:blank:]]*)[[:blank:]]*:`)
+
 	matches := subsystemPattern.FindStringSubmatch(subject)
-	if matches == nil {
-		return fmt.Errorf("Commit %v: Failed to find subsystem in subject: %q",
-			commit, subject)
+
+	var subsystem string
+
+	if matches == nil || len(matches) != 2 {
+		return fmt.Errorf("Commit %v: Failed to find subsystem in subject: %q", commit, subject)
 	}
 
+	// match 1 is the entire matching string
+	// match 2 is the subsystem name (without the colon)
+	subsystem = matches[1]
```

Comment:
accorinding to the comment, dont you need matches[2]?

Reply:
Good question. Those comments were supposed to refer to "1st" and "2nd", but I've changed it to make it clearer. Thanks!